python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1785,6 +1778,11 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
                 gains=gains,
             )
 
+        # Save the untrimmed version for later statistics,
+        # which still contains the overscan information
+        if self.config.doCalculateStatistics:
```

Comment:
Is this the right check?  It's not if we're calculating statistics at all, it's if we're computing CTI statistics.  As this is written we're going to be doing an expensive clone operation for regular ISR which we do not want.  So we want to check if we're doing cti statistics, `self.config.isrStatistics.doCtiStatistics` (I think?) and if not set `untrimmedCcdExposure = None` to protect it below.